actors/common/src/main/java/org/sunbird/metrics/actors/OrganisationMetricsActor.java

```diff
@@ -515,7 +528,9 @@ private void orgConsumptionMetrics(Request actorMessage) {
       if (StringUtils.isBlank(orgRootId)) {
         orgRootId = orgId;
       }
-      ProjectLogger.log("RootOrgId " + orgRootId);
+      ProjectLogger.log(
+          "OrganisationMetricsActor:orgConsumptionMetrics RootOrgId value is " + orgRootId,
```

Comment:
From:
"OrganisationMetricsActor:orgConsumptionMetrics RootOrgId value is "

To:
"OrganisationMetricsActor:orgConsumptionMetrics: root org id = "